File: Intermed/funson.cpp
#include <iostream>

int sub(int x, int y);

// Function to add two integers
int add(int a, int b) {
    return a + b;
}

int main() {
    int num1 = 4, num2 = 6;

    int sum = add(num1, num2);
    std::cout << "The sum is: " << sum << std::endl;
    std::cout << "The diff is: " 
              << sub(num1, num2) << std::endl;

    return 0;
}

int sub(int x, int y) {
    return abs(x - y);
}

File: Intermed/funson.py
def sub(x, y):
    return abs(x - y)

# Function to add two integers
def add(a, b):
    return a + b

def main():
    num1 = 4
    num2 = 6

    sum_val = add(num1, num2)
    print("The sum is:", sum_val)
    print("The difference is:", sub(num1, num2))

if __name__ == "__main__":
    main()
